ubuntu_image/common_builder.py

```diff
@@ -211,6 +211,27 @@ def pre_populate_bootfs_contents(self):
                 os.makedirs(target_dir, exist_ok=True)
         self._next.append(self.populate_bootfs_contents)
 
+    @classmethod
+    def _selective_copytree(cls, src, dst):
+        # Selectively copy entries from src to dst directory. Entries already
+        # existing at dst are not overwritten when overwrite is True.
+        os.makedirs(dst, exist_ok=True)
+        shutil.copystat(src, dst)
+        for entry in os.scandir(src):
+            sname = os.path.join(src, entry.name)
+            dname = os.path.join(dst, entry.name)
+
```

Comment:
In ubuntu-image, we have a very distinct coding 'style' in which there are no empty newlines between code. This is something Barry started and I'd like us to stay consistent. This basically means that if something should have some distinction to be more readable, one can put a comment with explanation there - otherwise, Python code itself is formatted well enough with whitespaces already.